ebu_tt_live/documents/test/converter_ericsson1.xml

```diff
@@ -12,7 +12,7 @@
                 <ebuttm:documentOriginalProgrammeTitle>tgv</ebuttm:documentOriginalProgrammeTitle>
                 <ebuttm:documentOriginalEpisodeTitle>tgv</ebuttm:documentOriginalEpisodeTitle>
                 <ebuttm:documentCreationDate>2016-09-06</ebuttm:documentCreationDate>
-                <ebuttm:documentRevisionNumber/>
+                <!-- <ebuttm:documentRevisionNumber/> -->
```

Comment:
Comment to say why this is here.

Reply:
I'll merge this similar change made elsewhere and resolve the clash so this goes away - I just repeated the fix here to get around the build error.